test/LeanCode.IntegrationTests/TimestampTzTests.cs

```diff
@@ -0,0 +1,67 @@
+using LeanCode.DomainModels.Model;
+using LeanCode.IntegrationTests.App;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace LeanCode.IntegrationTests;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("?", "CA1001", Justification = "Disposed with `IAsyncLifetime`.")]
+public class TimestampTzTests : IAsyncLifetime
+{
+    private readonly TestApp app;
+
+    public TimestampTzTests()
+    {
+        app = new TestApp();
+    }
+
+    [PostgresFact]
+    public async Task TimestampTz_local_timestamp_access_is_correctly_translated()
```

Comment:
Let's refactor this one, as this test isn't that self-explanatory.

From what I understand, it checks:
1) That the generated SQL has correct `ORDER BY` statement,
2) That the behavior of the query is correct, in two versions.

We should split that into two tests. Performance, for now (and I presume - for later also), is not really a problem and we can favor readability instead.

Also, let's remove from sight what doesn't really matter, mainly the "arrange" part that adds data to the database. And let's name the assertions, esp. the one that check SQL statement.

Also, Fluent :)

Reply:
> We should split that into two tests. Performance, for now (and I presume - for later also), is not really a problem and we can favor readability instead.

:+1:

> Also, let's remove from sight what doesn't really matter, mainly the "arrange" part that adds data to the database.

I can extract this to a separate method. Cannot really hide it though, it still needs to happen here.

> And let's name the assertions, esp. the one that check SQL statement.

How does one name method calls? Should I add labels, known from `goto`? Or would extracting them to their own methods be acceptable?

> Also, Fluent :)

:(

Reply:
> How does one name method calls? Should I add labels, known from goto? Or would extracting them to their own methods be acceptable?

Paraphrasing a classic, "Too much sarcasm will kill you" ;P

> I can extract this to a separate method. Cannot really hide it though, it still needs to happen here.

You will reduce it from tens (~20?) of lines to one, it will basically be hidden.